lesson08/index.js

```diff
@@ -0,0 +1,70 @@
+const imageArray = [
+    {
+        img: "1.png", 
+        alt:"画像1", 
+        text: "ブックマーク"
+    }, 
+    {
+        img: "2.png", 
+        alt:"画像2", 
+        text: "メッセージ"
+    }
+];
+
+const ul = document.getElementById('js-list');
+
+const createLoading = () => {
+    const li = document.createElement("li");
+    const loadingImg = document.createElement('img');
+
+    li.style.listStyle = "none";
+    li.id = "js-loading";
+    loadingImg.src = 'loading-circle.gif';
+    ul.appendChild(li).appendChild(loadingImg);
+};
+
+const removeLoading = () => {
+    const li = document.getElementById('js-loading');
+
+    ul.removeChild(li);
```

Comment:
IMO
I think you can also write it like this
```suggestion
    const li = document.getElementById('js-loading').remove();
```

Reply:
@sae-github good.

```js
document.getElementById('js-loading').remove();
```
just write this. because remove function is not return. li is undefined

Reply:
Thank you for review!
Changed the writing style.
[b843b2a](https://github.com/AyakaOnozuka131/JavaScript-lessons/pull/8/commits/b843b2a6fdd6c6a97025c6ccb61abd090d2337b3)

Reply:
My code is incorrect...
I'm sorry🙏

Please check @kenmori 's comment and correct it again.

Reply:
I'm sorry about that too.
We have decided that the relevant part is unnecessary and have deleted it.
Therefore, I will be careful next time.
Thank you for letting me know.
[161b612](https://github.com/AyakaOnozuka131/JavaScript-lessons/pull/8/commits/161b61260860a77cf54b6a8754e719ae1fa7b1c9#diff-dd20c42892cb73d7def3d5da99de657f43e811df6972d4c415de8603c7fa2c91L26-L28)